<launch>

    <node pkg="mqtt_node" type="vehicle_node.py" name="vehicle_node" output="screen"/>

</launch>